dbms/src/Flash/Statistics/JoinImpl.cpp

```diff
@@ -48,14 +43,6 @@ void JoinStatistics::collectExtraRuntimeDetail()
         build_side_child = join_execute_info.build_side_root_executor_id;
         is_spill_enabled = join_execute_info.join_ptr->isEnableSpill();
         is_spilled = join_execute_info.join_ptr->isSpilled();
-        for (const auto & non_joined_stream : join_execute_info.non_joined_streams)
```

Comment:
https://github.com/pingcap/tiflash/blob/ce52266d9d87c4b3b449d97a977ef0cfd0405fa3/dbms/src/Flash/Coprocessor/ExecutionSummaryCollector.cpp#L88-L109
Has this code been migrated?

Reply:
my mistake